test/unit/math/prim/meta/size_mvt_test.cpp

```diff
@@ -0,0 +1,26 @@
+#include <stan/math/prim/meta.hpp>
+#include <gtest/gtest.h>
+#include <vector>
+
+TEST(MathMetaPrim, size_mvt_scalar) {
+  using stan::size_mvt;
+
+  double x1;
+  EXPECT_THROW(size_mvt(x1), std::invalid_argument);
+
+  int x2;
+  EXPECT_THROW(size_mvt(x2), std::invalid_argument);
+}
+
+TEST(MathMetaPrim, size_mvt_matrices_vectors) {
+  using stan::size_mvt;
+
+  Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic> x1(2, 3);
+  EXPECT_EQ(1U, size_mvt(x1));
+
+  std::vector<Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic>> x2(3);
+  EXPECT_EQ(3U, size_mvt(x2));
+
+  std::vector<Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic>> x3(7);
+  EXPECT_EQ(7U, size_mvt(x3));
```

Comment:
This was pre-existing, but is the second of these really helpful? Or should one of these be an Eigen vector?

Reply:
Yep, good call. Changed this test to 

```cpp
TEST(MathMetaPrim, size_mvt_matrices_vectors) {
  using stan::size_mvt;

  Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic> x1(2, 3);
  EXPECT_EQ(1U, size_mvt(x1));

  std::vector<Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic>> x2(3);
  EXPECT_EQ(3U, size_mvt(x2));

  std::vector<Eigen::Matrix<double, 1, Eigen::Dynamic>> x3(7);
  EXPECT_EQ(7U, size_mvt(x3));

  std::vector<Eigen::Matrix<double, Eigen::Dynamic, 1>> x4(7);
  EXPECT_EQ(7U, size_mvt(x4));
}
```

Reply:
Nice!